recipe/meta.yaml

```diff
@@ -172,6 +182,16 @@ outputs:
         # GPU reuqirements
         - cudnn                   # [cuda_compiler_version != "None"]
         - nccl                    # [cuda_compiler_version != "None"]
+        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+        - cuda-cupti-dev    # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-cudart-dev   # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-nvml-dev     # [(cuda_compiler_version or "").startswith("12")]
+        - cuda-nvtx-dev     # [(cuda_compiler_version or "").startswith("12")]
+        - libcublas-dev     # [(cuda_compiler_version or "").startswith("12")]
+        - libcusolver-dev   # [(cuda_compiler_version or "").startswith("12")]
+        - libcurand-dev     # [(cuda_compiler_version or "").startswith("12")]
+        - libcufft-dev      # [(cuda_compiler_version or "").startswith("12")]
+        - libcusparse-dev   # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
Per https://github.com/conda-forge/tensorflow-feedstock/issues/296#issuecomment-1835781851, `cuda-nvcc-tools` should be installed during runtime

Reply:
Do we want `cuda-nvcc-impl` as well?

Asking as a complete NVVM install would use both. Though Idk which portions of NVVM are needed here

Reply:
FYI: I'm waiting until this is resolved with the builds. I didn't have time to start them over the weekend, so nothing has been built yet.

Reply:
This conda package (`cuda-nvcc-tools`) is only for CUDA 12. For CUDA 11 it's included in `cudatoolkit`.

Reply:
Done in commit ( https://github.com/conda-forge/tensorflow-feedstock/pull/353/commits/b9d069ca96e5990585a7642c29fc5ad24232e37b )